Backend/src/main/java/ro/uaic/info/romandec/repository/ManuscriptRepository.java

```diff
@@ -4,12 +4,20 @@
 import org.springframework.data.jpa.repository.Query;
 import ro.uaic.info.romandec.models.Manuscript;
 
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
 public interface ManuscriptRepository extends JpaRepository<Manuscript,Integer> {
```

Comment:
Annotate with `@Repository`
```suggestion
@Repository
public interface ManuscriptRepository extends JpaRepository<Manuscript,Integer> {
```